### PR TITLE
Add cache build flag

### PR DESCRIPTION
**Pull request by xumia**
**- What I did**
Add ENABLE_SYNCD_RPC for DPKG Cache

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: Makefile.cache [hunk 1]
```diff
@@ -69,7 +69,7 @@
 SONIC_COMMON_FILES_LIST :=  $(if $(wildcard cache.skip.common),, .platform slave.mk rules/functions Makefile.cache)
 SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(SONIC_DPKG_CACHE_SOURCE) $(SONIC_DEBUGGING_ON) \
-                            $(SONIC_PROFILING_ON) $(SONIC_SANITIZER_ON)
+                            $(SONIC_PROFILING_ON) $(SONIC_SANITIZER_ON) $(ENABLE_SYNCD_RPC)
```
Comment (qiluo-msft) [inline Makefile.cache]: >ENABLE_SYNCD_RPC [](start = 74, length = 16)

which one should use? SONIC_ENABLE_SYNCD_RPC?

Comment (qiluo-msft) [inline Makefile.cache]: >SONIC_SANITIZER_ON [](start = 52, length = 18)

This is nowhere used. Can we remove?

Comment (xumia) [reply Makefile.cache]: Fixed. Should follow the same way, using the variable starting with SONIC_

Comment (xumia) [reply Makefile.cache]: Removed.

**Comment by xumia** [comment]
retest vsimage please

**Pull request merged**